inference-engine/tests/functional/plugin/cpu/single_layer_tests/pad.cpp

```diff
@@ -4,57 +4,75 @@
+        float argPadValue;
         CPUSpecificParams cpuParams;
-        std::tie(basicParamsSet, cpuParams) = obj.param;
-
-        std::ostringstream result;
-        result << LayerTestsDefinitions::PadLayerTest::getTestCaseName(testing::TestParamInfo<LayerTestsDefinitions::padLayerTestParamsSet>(
-                basicParamsSet, 0));
-
-        result << CPUTestsBase::getTestCaseName(cpuParams);
-
-        return result.str();
+        std::tie(shapes, elementType, padsBegin, padsEnd, argPadValue, padMode, cpuParams) = obj.param;
+
+        std::ostringstream results;
+        results << "IS=" << CommonTestUtils::partialShape2str({shapes.first}) << "_";
+        results << "TS=";
+        for (const auto& item : shapes.second) {
+            results << CommonTestUtils::vec2str(item) << "_";
+        }
+        results << "Prc=" << elementType << "_";
+        results << "padsBegin=" << CommonTestUtils::vec2str(padsBegin) << "_";
+        results << "padsEnd=" << CommonTestUtils::vec2str(padsEnd) << "_";
+        if (padMode == ngraph::helpers::PadMode::CONSTANT) {
+            results << "Value=" << argPadValue << "_";
+        }
+        results << "PadMode=" << padMode << "_";
+        results << CPUTestsBase::getTestCaseName(cpuParams);
+
+        return results.str();
     }
 
 protected:
     void SetUp() override {
-        LayerTestsDefinitions::padLayerTestParamsSet basicParamsSet;
+        InputShape shapes;
+        ElementType elementType;
+        std::vector<int64_t> padsBegin, padsEnd;
+        ngraph::helpers::PadMode padMode;
+        float argPadValue;
         CPUSpecificParams cpuParams;
-        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+        std::tie(shapes, elementType, padsBegin, padsEnd, argPadValue, padMode, cpuParams) = this->GetParam();
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
-
-        InferenceEngine::SizeVector inputShape;
-        std::vector<int64_t> padsBegin, padsEnd;
-        float argPadValue;
-        ngraph::helpers::PadMode padMode;
-        InferenceEngine::Precision netPrecision;
-        std::tie(padsBegin, padsEnd, argPadValue, padMode, netPrecision, inPrc, outPrc, inLayout, inputShape, targetDevice) =
-                basicParamsSet;
-
-        inPrc = outPrc = netPrecision;
-        selectedType = std::string("ref_") + netPrecision.name();
-        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-        auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
-        auto paramOuts = ngraph::helpers::convert2OutputVector(
-                ngraph::helpers::castOps2Nodes<ngraph::opset3::Parameter>(params));
-        auto pad = ngraph::builder::makePad(paramOuts[0], padsBegin, padsEnd, argPadValue, padMode);
+        if (selectedType.empty()) {
+            selectedType = getPrimitiveType();
+        }
+        selectedType = selectedType + "_" + InferenceEngine::details::convertPrecision(inType).name();
```

Comment:
When https://github.com/openvinotoolkit/openvino/pull/8339 is merged you can use `makeSelectedTypeStr` utility function